### PR TITLE
Updated clair db import for Harbor 1.6+

### PR DESCRIPTION
**Pull request by SDBrett**
Updated documentation for importing vulnerability data into the clair DB. 

Reference issue: #6714 

Signed-off-by: Brett Johnson <brett@sdbrett.com>

### CODE REVIEW

**Comment by reasonerjt** [comment]
Thanks @SDBrett for the PR.

I don't think we need to keep the <= 1.6 as the doc should reflect the current version, for older version user can reference to the doc in other branches?

**Comment by SDBrett** [comment]
I'll clean up the pre 1.6 items and place a reference to previous versions of the document.

File: docs/import_vulnerability_data.md [hunk 1]
```diff
@@ -29,28 +29,39 @@ B. Check whether your Clair instance has already updated the vulnerability datab
     ```
 - The phrase "finished fetching" indicates that Clair has finished a round of vulnerability update from an endpoint. Please make sure all five endpoints (rhel, alpine, oracle, debian, ubuntu) are updated correctly. 
 
+## Harbor version < 1.6
+
+If you're using a version of Harbor prior to 1.6, you can access the correct instructions for your version using the following URL.
+https://github.com/goharbor/harbor/blob/v<VERSION NUMBER>/docs/import_vulnerability_data.md
```
Comment (ywk253100) [inline docs/import_vulnerability_data.md]: The link isn't shown correctly after the MD file is rendered.

**Comment by haisum** [comment]
These instructions are still invalid. Instead of referring to harbor-db, they refer to clair-db and the dump that they create in vulnerability.sql is only DDLs without any data.

**Pull request merged**